### PR TITLE
Added plane TPG threshold configurables.

### PR DESCRIPTION
**Pull request by aeoranday**
Configurables PR for TPG threshold by plane implementation in [fdreadoutlibs](https://github.com/DUNE-DAQ/fdreadoutlibs/pull/177). The `<plane>_threshold` default value of 0 makes that plane's threshold the same as `tpg_threshold` during runtime.

### CODE REVIEW

File: schema/readoutlibs/readoutconfig.jsonnet [hunk 1]
```diff
@@ -69,6 +69,12 @@ local readoutconfig = {
                             doc="Enable SimpleThreshold TPG algorithm only on collection planes when a Running Sum algorithm is enabled"),
             s.field("tpg_threshold", self.count, 120,
                             doc="Select TPG threshold"),
+            s.field("collection_threshold", self.count, 0,
```
Comment (alessandrothea) [inline schema/readoutlibs/readoutconfig.jsonnet]: `tpg_collection_threshold` maybe

File: schema/readoutlibs/readoutconfig.jsonnet [hunk 2]
```diff
@@ -69,6 +69,12 @@ local readoutconfig = {
...
+                            doc="Select TPG threshold for collection plane. Uses tpg_threshold if 0."),
+            s.field("induction1_threshold", self.count, 0,
```
Comment (alessandrothea) [inline schema/readoutlibs/readoutconfig.jsonnet]: `tpg_induction1_threshold`

File: schema/readoutlibs/readoutconfig.jsonnet [hunk 3]
```diff
@@ -69,6 +69,12 @@ local readoutconfig = {
...
+                            doc="Select TPG threshold for induction 1 plane. Uses tpg_threshold if 0."),
+            s.field("induction2_threshold", self.count, 0,
```
Comment (alessandrothea) [inline schema/readoutlibs/readoutconfig.jsonnet]: `tpg_induction2_threshold`

File: schema/readoutlibs/readoutconfig.jsonnet [hunk 4]
```diff
@@ -69,6 +69,12 @@ local readoutconfig = {
                             doc="Enable SimpleThreshold TPG algorithm only on collection planes when a Running Sum algorithm is enabled"),
             s.field("tpg_threshold", self.count, 120,
```
Comment (alessandrothea) [inline schema/readoutlibs/readoutconfig.jsonnet]: What is this used for? Remove?

**Review by alessandrothea — CHANGES_REQUESTED** [review]

File: schema/readoutlibs/readoutconfig.jsonnet (hunk 4 above)
Comment (aeoranday) [reply schema/readoutlibs/readoutconfig.jsonnet]: Plane thresholds that were not explicitly set default to `tpg_threshold` and keeps some backwards compatibility. It is easy enough to remove.

**Comment by aeoranday** [comment]
Latest commit made the word choice changes to match what is used in `detchannelmaps`. Configs and variables now refer to planes 0, 1, and 2 instead of induction 1, induction 2, and collection.

**Review by alessandrothea — APPROVED** [review]

**Pull request merged**